pkgs/desktops/budgie/budgie-desktop/default.nix

```
@@ -47,6 +49,20 @@ stdenv.mkDerivation rec {
   };
 
   patches = [
+    # Drop all Vapi files that aren't included with Vala
```

I prefer using the [PR title](https://github.com/BuddiesOfBudgie/budgie-desktop/pull/392/files) here, since the commit message [looks wrong](https://gitlab.gnome.org/GNOME/vala/-/tree/main/vapi) :joy: 

```suggestion
    # Drop all Vapi files that are already included with Vala
```